pkg/service/tools/tools.go

```diff
@@ -283,3 +287,113 @@ func (t *Tools) CreateConfig(_ context.Context, filePath string, configData stri
 	t.logger.Info("Config file generated successfully")
 	return nil
 }
+
+// Normalise initiates the normalise process for normalising the test cases.
+func (t *Tools) Normalise(_ context.Context) error {
```

Comment:
You should use yaml functions to edit the test-cases. You should how it is done in record and replay via interfaces of DBs . you can add this function in record or replay service.

Reply:
Added it to replay service